tests/factories/transaction.py

```diff
@@ -71,20 +71,19 @@ class Meta:
     expected_on = Faker("date_this_month")
 
     payment_type = "boleto"
```

Comment:
Pode remover esse daqui também. Se não for quebrar mais nenhum teste

```suggestion
    payment_type = "boleto"
```